blaze/utils.py

```diff
@@ -187,6 +192,11 @@ def json_dumps(ds):
     return {'__!datashape': str(ds)}
 
 
+@dispatch(types.BuiltinFunctionType)
```

Comment:
yep - this needs more work - just getting the conversation started 👍 

Thanks @llllllllll